docs/api/materials/MeshPhongMaterial.html

```diff
@@ -178,6 +178,13 @@ <h3>[property:Texture normalMap]</h3>
 			the way the color is lit. Normal maps do not change the actual shape of the surface, only the lighting.
 		</p>
 
+		<h3>[property:Integer normalMapType]</h3>
+		<p>
```

Comment:
Consider consolidating multiple document entries by moving the property to `Texture`. If not, add the documentation for `MeshNormalMaterial`.